test/babel-transform-undebug.mjs

```diff
@@ -0,0 +1,46 @@
+import path from 'path'
+import test from 'tape'
+import babel from '@babel/core'
+import transformConstants from '../script/babel-transform-undebug.mjs'
+
+function transformCode(code) {
+  const result = babel.transformSync(code, {
```

Comment:
maybe have a `configFile: false` here? https://babeljs.io/docs/en/options#configfile

Recently working on projects with babel, and it can pick up babel configs, which in this case it shouldn’t I believe?